<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlayWindow</class>
 <widget class="QMainWindow" name="OverlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>647</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>793</width>
    <height>647</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>793</width>
    <height>647</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnOpenBitmap">
        <property name="text">
         <string>Open Bitmap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenOverlay">
        <property name="text">
         <string>Open Overlay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerateOverlaid">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Generate Bitmap w/ Overlay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveBitmap">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Bitmap w/ Overlay</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Orginal Bitmap</string>
       </attribute>
       <widget class="QGraphicsView" name="gfxBitmap">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>571</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Overlay Bitmap</string>
       </attribute>
       <widget class="QGraphicsView" name="gfxOverlay">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>571</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Bitmap w/ Overlay</string>
       </attribute>
       <widget class="QGraphicsView" name="gfxOverlaid">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>571</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
